synapse/handlers/pagination.py

```diff
@@ -399,23 +435,49 @@ def get_delete_ids_by_room(self, room_id: str) -> Optional[StrCollection]:
         """
         return self._delete_by_room.get(room_id)
 
-    async def purge_room(self, room_id: str, force: bool = False) -> None:
+    async def purge_room(
+        self,
+        room_id: str,
+        delete_id: str,
+        force: bool = False,
+        shutdown_response: Optional[ShutdownRoomResponse] = None,
+    ) -> None:
         """Purge the given room from the database.
-        This function is part the delete room v1 API.
 
         Args:
             room_id: room to be purged
+            delete_id: the delete ID for this purge
             force: set true to skip checking for joined users.
+            shutdown_response: optional response coming from the shutdown phase
         """
+        logger.info("starting purge room_id %s", room_id)
```

Comment:
`force` seems like a detail that would be useful when log diving

```suggestion
        logger.info("starting purge room_id=%s force=%s", room_id, force)
```